elementary/monitor/api/models/schema.py

```diff
@@ -21,7 +21,7 @@ class NormalizedArtifactSchema(ExtendedBaseModel):
     # Currently, it's model_name to match the CLI UI.
     model_name: str
     normalized_full_path: str
-    fqn: str
+    fqn: Optional[str] = None
```

Comment:
I'm not sure if the FE actually relies on this field existing, worth double-checking

Reply:
It's used in catalog, and lineage's bottom drawer.
Tests don't appear in either of those so that's fine.